IFAS/DAL/HistoryDAL.py

```diff
@@ -0,0 +1,14 @@
+from db import db_connection
```

Comment:
you probably meant to delete this file (because you have both ifHistoryDAL and this one)